### DISCUSSION TITLE
雙螢幕小技巧詢問

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1777] [General]
最近剛買了一個螢幕，想知道有沒有辦法再用延伸功能時讓桌面圖示在兩邊都顯示然後在哪個螢幕按的應用程式就在哪個螢幕開啟，我昨天試了各種方法了都沒辦法，不然現在都是在主螢幕開啟有點不直覺QQ，不知道大家有沒有什麼小技巧

**Comment by poiu6fi** [comment]
我也有同樣的困擾，點開之後都會把視窗拉來拉去
